kotsadm/pkg/snapshot/backup.go

```diff
@@ -180,6 +182,7 @@ func CreateInstanceBackup(ctx context.Context, cluster *downstreamtypes.Downstre
 			// no version is deployed for this app yet
 			continue
 		}
+		appsSequences[a.Slug] = strconv.FormatInt(parentSequence, 10)
```

Comment:
It would be better not to use `FormatInt` here, and just make it a `map[string]int`.